WcaOnRails/spec/features/manage_competition_spec.rb

```diff
@@ -17,6 +20,7 @@
       fill_in "Name", with: "New Comp 2015"
 
       click_button "Create Competition"
+      expect(page).to have_content "Successfully created new competition!"
 
       expect(Competition.all.length).to eq 1
```

Comment:
That was not my understanding of @timhabermaas's comment [here](https://github.com/thewca/worldcubeassociation.org/pull/954#discussion_r83903305). The race was that we'd click the button and then immediately check the database... by waiting for the resulting page to load first, we know that the server has already written to the database before we go and look at the database.